### ISSUE TITLE
Weird/slow, behaviour on ssh login and http requests

### ISSUE THREAD

**Issue by hilja** [Varying-Vagrant-Vagrants/VVV#787]
WPML (multilingua plugin) has a way to use own domains per each language. It performs a domain validity check before saving the settings using [`WP_Http()`](https://developer.wordpress.org/reference/classes/wp_http/request/) function, in similar fashion as below:

``` php
$client   = new WP_Http();
$response = $client->request( 'http://local.wordpress.dev/', 'timeout=20' );
print_r($response);
```

If I run that, it will always timeout. It succeeds when using a "real" domain (like http://google.com).

If I run `$ curl -i http://local.wordpress.dev/` in the box, I gets the document (sometimes it's really slow, like ~30 seconds, but mostly it works).

Any idea why that `WP_Http()` fails?
### SSH

When I `$ vagrant ssh` it take a long time (~12 seconds) to connect. Is it normal? Not really a problem, but might be related...

**Comment by jeremyfelt** [comment]
Hi @hilja, thanks for the ticket and sorry for the lack of response.

I've run into this before when the VM's `/etc/hosts/` file uses the loopback address (127.0.0.1) for the domain name. I've fixed it by adjusting the hosts file to use `192.168.50.4` instead.

That said, I've also not done this and had no trouble at all, so I'll admit that I have no idea when it causes an issue and when it doesn't.

It would be nice to find the best way of managing this and make sure that `/etc/hosts` always contains the right info.

**Comment by tomjn** [comment]
I'm going to mark this as closed as nobody seems to have had the problem since Jeremeys changes
